src/components/calculator/PointsDisplay.tsx

```diff
@@ -78,77 +71,187 @@ export function ExplanationCard(props: {
+  { style: 'high', title: 'High', max: 300 },
+  { style: 'very-high', title: 'Very High', max: 1000 },
+  {
+    style: 'dangerous',
+    title: 'Dangerously High',
+    max: 100000,
+    icon: BsExclamationTriangleFill,
+  },
+]
+
+const RISK_LEVELS_TO_SHOW_ON_THERMOMETER = 5 // Shows up through 'Very High'
+
+/**
+ * Pick the top X risk levels for display and use reverse the order (so they display correctly)
+ */
+const riskLevelsForThermometer = riskLevels
+  .slice(0, RISK_LEVELS_TO_SHOW_ON_THERMOMETER)
+  .reverse()
+
+// const lifeThreateningRisk = {
+//   style: 'dangerous',
+//   title: 'Life-Threatening',
+//   max: 1e15, // Very large number
+//   icon: BsExclamationOctagonFill,
+// }
 
-function howRisky(points: number, budget: number): string[] {
+const howRisky = (points: number, budget: number): RiskLevel => {
+  // First check against dangerous risk levels. Don't normalize points here because we primarily want to indicate the risk to others, not the risk to you others at these "dangerous" levels
+  // TODO: Decide if we want to re-integrate this level later
+  const highestNormalRisklevel = riskLevels[riskLevels.length - 1]
+  // if (points >= highestNormalRisklevel.max) {
+  //   return lifeThreateningRisk
+  // }
+
+  // Then check against normalized points
   const normalizedPoints = points / (budget / 10000)
-  if (normalizedPoints < 3) {
-    return ['very low', riskyStyles[STYLE_LOW]]
-  } else if (normalizedPoints < 25) {
-    return ['low', riskyStyles[STYLE_LOW]]
-  } else if (normalizedPoints < 100) {
-    return ['moderate', riskyStyles[STYLE_MEDIUM]]
-  } else if (normalizedPoints < 300) {
-    return ['high', riskyStyles[STYLE_HIGH]]
-  } else if (normalizedPoints < 1000) {
-    return ['very high', riskyStyles[STYLE_HIGH]]
-  } else {
-    return ['dangerously high', riskyStyles[STYLE_HIGH]]
+  const curLevel = riskLevels.find((level) => normalizedPoints < level.max)
+  return (
+    curLevel || highestNormalRisklevel // Default to the highest risk level
+  )
+}
+
+const getWeekBudget = (budget: number) => {
+  return budget / 50 // Numbers look cleaner than 52.
+}
+
+const budgetConsumption = (points: number, budget: number) => {
+  const weekBudget = getWeekBudget(budget)
+  const weeksConsumed = points / weekBudget
+  if (weeksConsumed >= 1.5) {
+    return `
+        ${fixedPointPrecision(weeksConsumed)}x  your weekly risk budget`
   }
+  return `${fixedPointPrecision(
+    (points / weekBudget) * 100,
+  )}% of your weekly risk budget`
 }
 
-const budgetConsumption = (
-  points: number,
-  budget: number,
-  repeatedEvent: boolean,
-) => {
-  if (repeatedEvent) {
-    return `Having this interaction regularly would use up
-        ~${fixedPointPrecision(
-          ((points * 52) / budget) * 100,
-        )}% of your annual risk
-        allocation.`
+function Thermometer(props: {
+  doShowPoints: boolean
+  activeRiskLevel: RiskLevel
+}): React.ReactElement {
+  const onActiveLevel = (
```

Comment:
I don't think this function makes for clearer code. Just inline it below.

Reply:
Actually, since currentRisk level is literally a pointer into riskLevels, it should be fine to do
`currentRiskLevel === comparisonRiskLevel`

Reply:
Done